### DISCUSSION TITLE
What does the 'ghost' icon imply?

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#7105] [Q&A]
Hello,

Trying to add some languages, Weblate crashed with an error. But what I'm wondering: since the crash I see a little ghost icon behind the languagages, and the alt text says "2 strings are not being translated here".
![Schermafdruk van 2022-01-15 17-50-14](https://user-images.githubusercontent.com/11229646/149630494-35ae108b-0ed2-4bb3-b0d4-9e9786fbd15d.png)

However, when accessing the individual components I don't see the ghost icon. Also I'm not sure what 'not being translated' implies: I suppose it's a bad thing, but what does it mean? Are there two source strings that are excluded somehow? Is something wrong with two strings? ...

Many thanks!
Cheers

**Comment by nijel** [comment]
This is probably a result of glossary entries being different for each language...

**Reply by keunes** [reply]
Ha, thanks. I hadn't set glossary terms and wasn't familiar yet with this area. I see that our French translator added some. Totally unrelated to the crash when attempting to add languages, then.

I see that `read-only` glossary terms indicate they cannot be translated (like brand names). In our glossary, a term inherited this flag from "AntennaPod/AntennaPod — English". But I think I made English read-only for the 'real' components to make sure edits arrive at the right place. But this should not be the case for the glossary. Is there a way to turn this off somehow? I can't seem to edit this glossary term (because of the read-only flag I presume).

**Reply by nijel** [reply]
You can remove the read-only flag (from the *Tools* menu).

**Reply by keunes** [reply]
That option doesn't seem to be available:
![Schermafdruk van 2022-01-19 09-02-36](https://user-images.githubusercontent.com/11229646/150089093-02e5a120-c0d8-4f27-9ca3-626738605002.png)

https://hosted.weblate.org/translate/antennapod/glossary/en/?checksum=1605c4924dcd03c7&sort_by=source

**Reply by keunes** [reply]
Could it be because I've marked it as [terminology](https://docs.weblate.org/en/latest/user/glossary.html#terminology) (meaning that the string gets added to all language's glossary)?

**Reply by keunes** [reply]
I tested a bit, and it seems that
* creating a new glossary term and keeping the checkbox 'terminology' ticked, leaves the term editable
* creating a new glossary term and turning off the 'terminology checkbox, later marking the string as terminology via the Tools menu, makes the term un-editable.

So behaviour is slightly inconsistent here.
